projects/addon-commerce/utils/test/get-currency-by-code.spec.ts

```diff
@@ -0,0 +1,23 @@
+import {
+    TUI_CURRENCY_DICTIONARY,
+    TuiCurrency,
+    TuiCurrencyCode,
+    tuiGetCurrencyByCode,
+} from '@taiga-ui/addon-commerce';
+
+describe(`tuiGetCurrencyByCode`, () => {
+    it(`AED`, () => {
+        expect(tuiGetCurrencyByCode(TuiCurrencyCode.Dirham)).toEqual(TuiCurrency.Dirham);
+        expect(tuiGetCurrencyByCode(TuiCurrencyCode.Dirham)?.toString()).toEqual(`AED`);
+    });
+
+    it(`the same number of keys`, () => {
+        expect(Object.keys(TUI_CURRENCY_DICTIONARY).length).toBe(23);
+
+        // @ts-ignore
+        expect(Object.keys(TuiCurrencyCode ?? {}).length).toBe(23);
+
+        // @ts-ignore
+        expect(Object.keys(TuiCurrency ?? {}).length).toBe(23);
```

Comment:
maybe `expect(Object.keys(TuiCurrency ?? {}).length).toBe(Object.keys(TuiCurrencyCode ?? {}).length)` ?

(btw first test seems useless - there is no logic and we test how js the object works)

Reply:
fixed